### PR TITLE
Add a caching warning

### PR DESCRIPTION
**Pull request by bobjflong**

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -29,6 +29,8 @@ Installing this plugin provides a new Intercom settings page, which allows you t
 
 If `current_user` is present, their email will be used as an identifier in the widget. Otherwise, it falls back to anonymous chat using [Acquire](https://www.intercom.io/live-chat).
 
+NB: This plugin injects a Javascript snippet on your website frontend containing dynamic user data. Some caching solutions will cache entire pages and should not be used with this plugin.
```
Comment (RuairiK) [inline readme.txt]: Perhaps we should explicitly explain that it may show conversations to other users?

Comment (bobjflong) [inline readme.txt]: I don't really know how to phrase that

Comment (RuairiK) [inline readme.txt]: How about:
> This plugin injects a Javascript snippet on your website frontend containing dynamic user data. Some caching solutions will cache entire pages and should not be used with this plugin. Doing so may cause conversations to be delivered to the wrong user.

File: readme.txt [hunk 2]
```diff
@@ -25,10 +25,12 @@ Full installation and usage instructions are available on the [Intercom website]
 
 With the Intercom plugin for WordPress, you can chat to both logged-in and anonymous users (using [Acquire](https://www.intercom.io/live-chat)).
 
-Installing this plugin provides a new Intercom settings page, which allows you to configure your app id and secure mode secret. Once filled out, the Intercom widget will automatically appear.
+This plugin injects a Javascript snippet on your website frontend containing dynamic user data. Some caching solutions will cache entire pages and should not be used with this plugin. Doing so may cause conversations to be delivered to the wrong user.
```
Comment (RuairiK) [inline readme.txt]: Did you mean to replace this line with the warning?

**Comment by RuairiK** [comment]
👍

**Pull request merged**